src/main/resources/application.conf

```
@@ -0,0 +1,65 @@
+ot {
+  common {
+    output = "gs://ot-snapshots/etl-2004"
```

We need to reduce hardcoded things like this. A good way of replacing this with something more appropiate could be using env vars
```suggestion
    output = "gs://ot-snapshots/etl/latest"
    output = ${?OT_ETL_OUTPUT}
```
So, in the case that variable is not declared the default string will be used in its case. This can also be combined with commandline arguments to overwrite the output. By example, I would be passing a conf file from url as jvm `-D` option as [you can see here](https://github.com/lightbend/config#standard-behavior). So we provide this internal application.conf as it is with some sane default as etl/latest and then we use a configuration from jvm arguments to provide a config from an url with just the changes we need if that makes sense like this way `-Dconfig.file=/srv/app/production.conf` or using `-Dconfig.url...` although currently we can leave like that as `etl/latest/`